feat: StatusBar extensions (#265)
unoplatform/uno.toolkit.ui: doc/helpers/StatusBar-extensions.md

# StatusBar (attached properties)
Provides two attached properties on `Page` to controls the visual of the status bar on mobile platforms.

## Remarks
The attached properties does nothing on platforms other than iOS and Android.
For iOS, `UIViewControllerBasedStatusBarAppearance` should set to false in `info.plist`.

## Properties
Property|Type|Description
-|-|-

> ForegroundTheme|StatusBarTheme|Sets the foreground color for the text and icons on the status bar. Possible values are: `None`, `Light`, `Dark`.

Reviewer: It's difficult to understand the meaning of those values. What does the `None` means? (no status bar??) If I put `Dark`, does that means the foreground color will be dark or it will be an appropriate color for a dark background?

Author: None, it is supposed to unsubscribe the event. I have no found a way to reset the color / knowing the initial color, so it kind of just leave it on the last set color. 
Dark means dark. The behavior you described would be more like "Auto".

> # StatusBar (attached properties)

Reviewer 2: ```suggestion
# StatusBar Attached Properties
```

> Provides two attached properties on `Page` to controls the visual of the status bar on mobile platforms.

Reviewer: This page is not referenced in the toc.yml ?

## Remarks

> The attached properties does nothing on platforms other than iOS and Android.

Reviewer 2: ```suggestion
The attached properties do nothing on platforms other than iOS and Android.
```

For iOS, `UIViewControllerBasedStatusBarAppearance` should set to false in `info.plist`.

## Properties
Property|Type|Description
-|-|-
ForegroundTheme|StatusBarTheme\*|Sets the foreground color for the text and icons on the status bar. Possible values are: `None, Light, Dark, Auto or AutoInverse`.
Background|Brush|Sets the background color for the status bar. <br/> note: Due to platform limitations, only `SolidColorBrush` are accepted.

> StatusBarTheme\*: `Auto` and `AutoInverse` will set the foreground accordingly to the theme, and update itself when the os theme or the app theme changes.

Reviewer 2: ```suggestion
StatusBarTheme\*: `Auto` and `AutoInverse` will set the foreground in accordance to the theme, and update itself when the system theme or the app theme changes.
```
